language/stdlib/modules/association.move

```diff
@@ -109,6 +114,156 @@ module Association {
     acquires PrivilegedCapability {
         Transaction::assert(addr_is_association(addr), 1002);
     }
-}
 
+// **************** SPECIFICATIONS ****************
+// Note: I experimented with using Markdown to organize this, but I haven't converted
+// it to work with the new doc generator, yet.
+
+// # Specifications for association.move
+
+// This is preliminary.  This is one of the first real module libraries with global
+// specifications, and everything is evolving.
+
+    spec module {
+        pragma verify = true;
+
+        // This mirrors the Move function root_address()
```

Comment:
The Google doc is good (I am going to check internally if we have our own guidelines written somewhere).

Maybe this has already been discussed at some other time - I am curious why we are not following the [Rust Documentation Comment format and policies](https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#making-useful-documentation-comments), given how much relationship there is to Move and Rust.

Re: referring to another function - generally, yes, the function should be public if the documentation referencing it is public. And you could use [xxx](LINK) to point to the file that would be created when generating the documentation for that function.

Reply:
Ok, in this case I want to say it's the same as root_address, because results will be confusing if they diverge. I spelled out that it must be consistent. We don't have to have the source code for root_address in front of us to understand it.  I changed "This".